### PR TITLE
Active fund note 2nd step

### PR DESCRIPTION
**Pull request by nickymiron**
active-fund-note

### CODE REVIEW

File: src/steps/transferFutureCapital/TransferFutureCapital.js [hunk 1]
```diff
@@ -50,7 +51,22 @@ export const TransferFutureCapital = ({
         className="mt-4"
         onSelect={() => onSelectFutureCapitalFund(null)}
       >
-        <p className="m-0"><Message>transfer.future.capital.no</Message></p>
+        <p className={`m-0 ${!selectedFutureContributionsFundIsin ? 'text-bold' : ''}`}>
+          <Message>transfer.future.capital.no</Message>
+        </p>
+        {
+          (!selectedFutureContributionsFundIsin && activeSourceFund) ?
+          (<p className="mb-0 mt-2">
```
Comment (Tankenstein) [inline src/steps/transferFutureCapital/TransferFutureCapital.js]: No need for parentheses here, let's remove 'em

File: src/steps/transferFutureCapital/TransferFutureCapital.spec.js [hunk 2]
```diff
@@ -59,7 +59,7 @@ describe('Transfer future capital step', () => {
       .get(0)).toEqual(<Message>transfer.future.capital.no</Message>);
   });
 
-  it('can where you want to transfer future capital', () => {
+  it('can chose where you want to transfer future capital', () => {
```
Comment (Tankenstein) [inline src/steps/transferFutureCapital/TransferFutureCapital.spec.js]: Typo, `choose`

File: src/translations/translations.et.json [hunk 3]
```diff
@@ -65,6 +65,7 @@
   "transfer.future.capital.AE123232334.fund": "Tuleva maailma aktsiate pensionifond",
   "transfer.future.capital.AE123232335.fund": "Tuleva maailma võlakirjade pensionifond",
   "transfer.future.capital.no": "Ei suuna makseid Tulevasse",
+  "transfer.future.capital.no.subtitle": "Sinu pensionimaksed lähevad ka edaspidi {{currentFundName}} pensionifondi, mille valitsemistasu on {{currentFundManagementFee}}% aastas. Kui soovid teenusasudelt säästa, saad oma maksed ka hiljem Tulevasse ümber suunata.",
```
Comment (Tankenstein) [inline src/translations/translations.et.json]: Typo, `teenustasudelt` not `teenusasudelt`

File: src/steps/transferFutureCapital/TransferFutureCapital.js (hunk 1 above)
Comment (nickymiron) [inline src/steps/transferFutureCapital/TransferFutureCapital.js]: okay

File: src/steps/transferFutureCapital/TransferFutureCapital.spec.js (hunk 2 above)
Comment (nickymiron) [inline src/steps/transferFutureCapital/TransferFutureCapital.spec.js]: uku

Damn...

File: src/translations/translations.et.json (hunk 3 above)
Comment (nickymiron) [inline src/translations/translations.et.json]: eestikeel

**Pull request merged**